src/mlte/value/types/real.py

```diff
@@ -140,18 +140,18 @@ def greater_than(self, value: float) -> ValidationResult:
         )(self)
         return result
 
-    def greater_or_equal_to(self, value: float) -> ValidationResult:
+    def greater_or_equal_to(self, value: float) -> Result:
```

Comment:
I believe you intend to address this in a separate PR, but we decided that these methods will return thunks rather than the result of evaluation itself, correct?

Reply:
Yes, that will be the case in the next PR.

Reply:
Resolved.